### PR TITLE
Feature/ansonlee99/983740187 add item button

### PR DESCRIPTION
**Pull request by AnsonLee99**
### Administrative Info
Monday Board ID: 983740187
Make sure your branch name conforms to: <feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Added "Add Item" button
- Added "Add Item form"
- Form sends POST request to backend

### Testing
How did you confirm your changes worked? 
- Visual check of form, inputted different values
- Connected to mongoDB and checked items were correctly added to the database

### Confirmation of Change 
image

### CODE REVIEW

File: client/src/add_button/AddButton.js
Comment (justinyaodu) [inline client/src/add_button/AddButton.js]: Should this variable be named `setOutByDate` instead of `setoutByDate`?

File: client/src/add_button/AddButton.js
Comment (justinyaodu) [inline client/src/add_button/AddButton.js]: I believe "e.g." should have another period between the "e" and the "g". It might also look nicer with the "e" uncapitalized, since the placeholder text is more of a phrase than a sentence, but I'll leave that up to you.

File: client/src/add_button/AddButton.js
Comment (justinyaodu) [inline client/src/add_button/AddButton.js]:
```suggestion
                        <NumericInput min={0} size={1} value={lbPerHouse} onChange={val => setLbPerHouse(val)}/>
```

File: client/src/MongodbFunctions.js
Comment (justinyaodu) [inline client/src/MongodbFunctions.js:2]: This URL will eventually need to be updated, right? I'm not sure what the best way to handle this is, though. Perhaps a variable somewhere which can point to localhost for now, and be changed to point to our deployment later?

File: client/src/add_button/AddButton.js
Comment (justinyaodu) [inline client/src/add_button/AddButton.js]: I'm not too familiar with JSX, so I'm not sure if this is valid syntax. In general though, when an arrow function only returns a value, the curly braces and return can be replaced with the expression for the return value.

```suggestion
                        <div>
                            Recommendations
                            {(rec.length === 0)
                                ? <p className ="recommendation_pos">No Issues</p>
                                : rec.map(txt=>{return <p className="recommendations_neg">{txt}</p>})
```

**Review by justinyaodu — COMMENTED** [review]
Looks good overall! I'd recommend testing it again after #7 is merged though, just to make sure everything still works.

File: client/src/MongodbFunctions.js
Comment (AnsonLee99) [reply client/src/MongodbFunctions.js:2]: I have all the HTTP requests related to MongoDB in the MongodbFunctions.js file, so all the changes related to the endpoint URL will be in one centralized place.

File: client/src/add_button/AddButton.js
Comment (AnsonLee99) [reply client/src/add_button/AddButton.js]: good catch! fixed!

File: client/src/add_button/AddButton.js
Comment (AnsonLee99) [reply client/src/add_button/AddButton.js]: updated!

File: client/src/add_button/AddButton.js
Comment (AnsonLee99) [reply client/src/add_button/AddButton.js]: oh yeah, forgot about that syntax. Definitely cleaner though!

File: client/src/add_button/AddButton.js
Comment (AnsonLee99) [reply client/src/add_button/AddButton.js]: fixed!

**Review by hahsu — CHANGES_REQUESTED** [review]
Nice job! Is it possible to move the lbs per household a bit to the right? Right now it's really close to the input box, can we add some whitespace between the input box and the text?

**Review by hahsu — APPROVED** [review]
lgtm!

**Review by justinyaodu — APPROVED** [review]

**Pull request merged**
